projects/packages/blaze/src/class-blaze.php

```diff
@@ -57,6 +61,27 @@ public static function add_post_links_actions() {
 		}
 	}
 
+	/**
+	 * Enable the Blaze menu.
+	 *
+	 * @return void
+	 */
+	public static function enable_blaze_menu() {
+		if ( self::should_initialize() ) {
```

Comment:
Maybe we should add a filter to gate the feature until it's ready?

Reply:
This is a good idea! I don't know if we can add FF or something here to close the functionality until is ready

Reply:
👍 I've just pushed 4f2d68ed7d10391f1529dfff4917428cb93ae7d3 to gate access to the feature for now.

Reply:
We may need to roll back the entry points I updated or add the same gate so you don't get redirected to the module if it is not opened.

I will try to make that change, it should be easy in PHP (I can use the same code as yours), but I don't know if we can use the same code in JS. I will try and see if I find any examples.

Reply:
I've just pushed a few commits to make that happen. Let me know what you think about it.

Ideally I would have preferred to build the full management link in one place instead of 2 (we now build it in php and in js), but the dynamic post ID would make this less clean in my opinion.